feed-formats/buttwoo.js

```diff
@@ -15,6 +15,13 @@ function base64ToBuffer(str) {
 
 const BUTTWOO_FEED_TF = bfe.toTF('feed', 'buttwoo-v1')
 
+function hashToBFE(buffer) {
+  return Buffer.concat([
+    Buffer.from([0]),
+    buffer
+  ])
+}
```

Comment:
I think the reason why I removed this is that I didn't understand this. It's not really BFE, is it? It's missing the `F` byte, and the `T` byte is zero, which means "feed". Why do we need this, and couldn't we add new hash functions as `buttwoo-v2`, `buttwoo-v3` etc?

Reply:
We could, but added buttwoo-v2 support requires changes something like 10 libraries. But yes, the name is super bad, I'll that.

Reply:
And that is only on the JS side :)